### PR TITLE
feat:update RolePermissions css

### PR DESCRIPTION
**Pull request by kogel-net**
# Description

_Please explain the changes you've made_

## Issue reference

We strive to have all PR being opened based on an issue, where the problem or feature have been discussed prior to implementation.

Please reference the issue this PR will close: #_[issue number]_

## Checklist

Please make sure you've completed the relevant tasks for this PR, out of the following list:

* [ ] Code compiles correctly
* [ ] Created/updated tests
* [ ] Extended the documentation

### CODE REVIEW

**Comment by doddgu** [comment]
@MayueCif PTAL

File: src/Web/Masa.Auth.Web.Admin.Rcl/Pages/RolePermissions/Permissions/AddMenuPermission.razor [hunk 1]
```diff
@@ -9,7 +8,7 @@
                         <DccI18nConfiger @bind-Name="_menuPermissionDetailDto.Name" />
                     </MCol>
                     <MCol Md=12>
-                        <STextField Label="@T("Code")" Required Prefix="@($"{_menuPermissionDetailDto.AppId}.")" @bind-Value="_menuPermissionDetailDto.Code" Messages='new List<string>{T("PermissionCodeMsg")}' />
+                    <STextField Label="@T("Code")" Required Prefix="@($"{_menuPermissionDetailDto.AppId}.")" @bind-Value="_menuPermissionDetailDto.Code" Messages='new List<s{("PermissionCodeMsg")}'Class="text-field" />
```
Comment (MayueCif) [inline src/Web/Masa.Auth.Web.Admin.Rcl/Pages/RolePermissions/Permissions/AddMenuPermission.razor:11]: ?

File: src/Web/Masa.Auth.Web.Admin.Rcl/Pages/RolePermissions/Permissions/AddMenuPermission.razor [hunk 2]
```diff
@@ -29,7 +28,7 @@
                         <STextField Label="@T("Icon")" Placeholder="@T("Support mdi and fas icon")" @bind-Value="_menuPermissionDetailDto.Icon">
                             <PrependContent>
                                 <div class="d-flex align-center mt-n1">
-                                    <MIcon Size=30>
+                                    <MIcon Size=30 Style="padding-top: 6px;">
```
Comment (MayueCif) [inline src/Web/Masa.Auth.Web.Admin.Rcl/Pages/RolePermissions/Permissions/AddMenuPermission.razor:31]: delete style

File: src/Web/Masa.Auth.Web.Admin.Rcl/Pages/RolePermissions/Permissions/AddMenuPermission.razor [hunk 3]
```diff
@@ -29,7 +28,7 @@
                         <STextField Label="@T("Icon")" Placeholder="@T("Support mdi and fas icon")" @bind-Value="_menuPermissionDetailDto.Icon">
                             <PrependContent>
                                 <div class="d-flex align-center mt-n1">
```
Comment (MayueCif) [inline src/Web/Masa.Auth.Web.Admin.Rcl/Pages/RolePermissions/Permissions/AddMenuPermission.razor:30]: remove mt-n1

File: src/Web/Masa.Auth.Web.Admin.Rcl/Pages/RolePermissions/Permissions/AddMenuPermission.razor [hunk 4]
```diff
@@ -1,5 +1,4 @@
 ﻿@inherits AdminCompontentBase
-
 <SSheetDialog @bind-Value=_visible Title="@T("Add Menu Permission")">
```
Comment (MayueCif) [inline src/Web/Masa.Auth.Web.Admin.Rcl/Pages/RolePermissions/Permissions/AddMenuPermission.razor:2]: alse update AddApiPermission and UpdateXXXPermission

File: src/Web/Masa.Auth.Web.Admin.Rcl/Pages/RolePermissions/Roles/UpdateRoleDialog.razor [hunk 5]
```diff
@@ -6,8 +6,8 @@
         <AutoHeight Overflow AutoClass="mt-6">
             <HeaderContent>
                 <div class="d-flex">
-                    <div class="mx-auto">
-                        <ButtonGroup @bind-Value="Tab" TValue="UpdateRoleTabs" />
+                    <div class="mx-auto d-role-tab">
```
Comment (MayueCif) [inline src/Web/Masa.Auth.Web.Admin.Rcl/Pages/RolePermissions/Roles/UpdateRoleDialog.razor:9]: d-role-tab

**Review by MayueCif — COMMENTED** [review]
@kogel-net

**Review by MayueCif — APPROVED** [review]
LGTM

**Pull request merged**